test/server/cards/02.1-ToA/KaradaDistrict.spec.js

```diff
@@ -124,6 +125,8 @@ describe('Karada District', function() {
 
                     expect(this.watchCommander.controller).toBe(this.player2.player);
                     expect(this.miyaMystic.attachments.toArray()).toContain(this.watchCommander);
+                    expect(this.getChatLogs(2)).toContain('player2 uses Karada District, giving 1 fate to player1 to choose a target for Watch Commander');
+                    expect(this.getChatLogs(1)).toContain('player2 chooses to attach Watch Commander to Miya Mystic');
                 });
             });
 
```

Comment:
If this is passing tests, we presumably don't have a test checking that you can move an attachment you control, would it be worth adding one?

Reply:
We do have a test, but as discussed, but Karada cannot move an attachment you control (it is just discarded).